### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by wallet77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,316 @@
...
+```
+
+## Report Caught Exceptions
+
+By default, in the Issue tab, you are only alerted for uncaught exceptions. Any exception that you catch is not reported. You can manually report them with the `notify()` method.
+
+```javascript
+const io = require('pm2.io');
+
+io.notify({ success: false });
```
Comment (vmarchaud) [inline README.md]: deprecated, it should be `notifyError` now

File: README.md [hunk 2]
```diff
@@ -16,8 +16,316 @@
...
+
+io.notify('This is an error');
```
Comment (vmarchaud) [inline README.md]: same as above

File: README.md [hunk 3]
```diff
@@ -16,8 +16,316 @@
...
+
+io.notifyError(new Error('This is an error'));
+```
+
+## Report Custom Events (deprecated)
```
Comment (vmarchaud) [inline README.md]: do we want to document them ? i don't see the point since they will be removed

File: README.md (hunk 1 above)
Comment (wallet77) [reply README.md]: Done, I remove documentation about notify because it's deprecated.

File: README.md (hunk 2 above)
Comment (wallet77) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (wallet77) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -16,8 +16,293 @@
...
+### Simple actions
+
+The function takes a function as a parameter (reply here) and need to be called once the job is finished.
+
+Example:
+
+```javascript
+const io = require('pm2.io');
+
+io.action('db:clean', (reply) => {
```
[the remark this answers is not in the document]
Comment (wallet77) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 <div align="center">
-  <a href="http://pm2.keymetrics.io">
+  <a href="http://@pm2/io">
```
[the remark this answers is not in the document]
Comment (wallet77) [reply README.md]: Done

File: README.md [hunk 6]
```diff
@@ -3,7 +3,7 @@
     <img width=411px src="https://raw.githubusercontent.com/keymetrics/pmx/master/pres/logo.png">
```
Comment (vmarchaud) [inline README.md]: need to change these links also

File: README.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
...
   </a>
   <br/>
-  <b><a href="https://github.com/Unitech/pm2">PM2</a> programmatic integration</b>
+  <b><a href="https://github.com/Unitech/pm2">PM2</a> monitoring module</b>
   <br/>
   <br/>
   <a href="https://www.bithound.io/github/keymetrics/pmx">
```
Comment (vmarchaud) [inline README.md]: we can remove this one i think

File: README.md (hunk 6 above)
Comment (wallet77) [reply README.md]: Done

**Pull request merged**